packages/frontend-plugin-api/src/types.ts

```diff
@@ -25,26 +25,20 @@ import { ComponentType, PropsWithChildren } from 'react';
 export type Expand<T> = T extends infer O ? { [K in keyof O]: O[K] } : never;
 
 /** @public */
-export type CoreProgressComponent = ComponentType<PropsWithChildren<{}>>;
+export type CoreProgressProps = PropsWithChildren<{}>;
```

Comment:
Tbh I think we should avoid `PropWithChildren` and then just define children as `children?: ReactNode` where needed, even without the `?` if children are required. In this case we don't want any children at all.

Reply:
Done 🙌🏻